app/models/piece.rb

```diff
@@ -86,32 +86,33 @@ def obstructed?(row_dest, col_dest)
         row_pos += 1
         (row_pos...row_dest).each do |row_num|
           col_pos += 1
-          return !game.pieces.find_by(captured: false, row_position: row_num, col_position: col_pos).nil?
+          return true unless game.pieces.find_by(row_position: row_num, col_position: col_pos).nil?
         end
       # row and col both decrease
       elsif row_pos > row_dest && col_pos > col_dest
         row_dest += 1
         (row_dest...row_pos).each do |row_num|
           col_dest += 1
-          return !game.pieces.find_by(captured: false, row_position: row_num, col_position: col_dest).nil?
+          return true unless game.pieces.find_by(row_position: row_num, col_position: col_dest).nil?
         end
       # row increases col decreases
       elsif row_pos < row_dest && col_pos > col_dest
         row_pos += 1
         (row_pos...row_dest).each do |row_num|
           col_pos -= 1
-          return !game.pieces.find_by(captured: false, row_position: row_num, col_position: col_pos).nil?
+          return true unless game.pieces.find_by(row_position: row_num, col_position: col_pos).nil?
         end
       # row decreases col increases
       else
         row_dest += 1
         (row_dest...row_pos).each do |row_num|
           col_dest -= 1
-          return !game.pieces.find_by(captured: false, row_position: row_num, col_position: col_dest).nil?
+          return true unless game.pieces.find_by(row_position: row_num, col_position: col_dest).nil?
         end
       end
     else
       return "ERROR! in is_obstructed? method line:83"
```

Comment:
I'm not sure I understand this message.  And line 83 is a comment, currently.  Maybe @addstar34 can explain what this was originally supposed to do.